### ISSUE TITLE
FBT003: ruff reports about function calls from C-extentions

### ISSUE THREAD

**Issue by Paciupa** [astral-sh/ruff#9287]
In my case this is calls from `pygame`, for example:
```python
pygame.mouse.set_visible(False)
```
Ruff reports `FBT003 Boolean positional value in function call`.
If I try use keyword argument, I'll get ```TypeError: set_visible() takes no keyword arguments```
And it happens with every function from every C-library that takes bool.
So the only solution now is suppress this message.
Ruff version: 0.1.9.

**Comment by charliermarsh** [comment]
I believe this is a duplicate of https://github.com/astral-sh/ruff/issues/3247. There aren't many good solutions here -- I typically recommend just not using the `FBT` rules (which are quite rigid and opinionated) if you're not working in a fairly strict project or setting.

**Comment by Paciupa** [comment]
Ok, thanks
